### PR TITLE
Fixed "[+&#124;-]"→"[+\|-]"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/dotnet/visual-basic/reference/command-line-compiler/compiler-options-listed-alphabetically

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
I think it must be escaped.

File: docs/visual-basic/reference/command-line-compiler/compiler-options-listed-alphabetically.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The Visual Basic command-line compiler is provided as an alternative to compilin
 |[-optioninfer](../../../visual-basic/reference/command-line-compiler/optioninfer.md)|Enables the use of local type inference in variable declarations.|  
 |[-optionstrict](../../../visual-basic/reference/command-line-compiler/optionstrict.md)|Enforces strict language semantics.|  
 |[-out](../../../visual-basic/reference/command-line-compiler/out.md)|Specifies an output file.|  
-|`-parallel[+&#124;-]`|Specifies whether to use concurrent build (+).|  
+|`-parallel[+\|-]`|Specifies whether to use concurrent build (+).|
```
Comment (mairaw) [inline docs/visual-basic/reference/command-line-compiler/compiler-options-listed-alphabetically.md]: That didn't work. Let's try this instead.

```suggestion
|<code>-parallel[+&#124;-]</code>|Specifies whether to use concurrent build (+).|  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @hyoshioka0128 for the bug fix but that didn't quite work. I left a suggestion on how I think we can make that work instead.

**Review by mairaw — APPROVED** [review]
I've applied the feedback so we can merge this PR if build looks good. We're starting to get customer feedback on this issue, so I'd like to merge this sooner rather than later.

**Pull request merged**